Cliqz/MyOffrz/OffrzDataSource.swift

```diff
@@ -7,26 +7,25 @@
 //
 
 import Foundation
+import RxSwift
 
 class OffrzDataSource {
 	
 	private var currentOffr: Offr?
 	private static let LastSeenOffrID = "LastSeenOffrID"
 	private static let OffrzOnboardingKey = "OffrzOnboardingNeeded"
+    let observable = BehaviorSubject(value: false)
 
 	var myOffrz = [Offr]()
-
-	init() {
-		self.updateMyOffrzList()
```

Comment:
the problem is that init is called before the view is created because the datasource is passed in the constructor of the view, which mean it could happen due to racing condition that the API return data before the view is fully drawn which might have some problems

Reply:
done